content/docs/notebooks/setup.md

```diff
@@ -126,25 +126,65 @@ When Kubeflow is running, you can access the Kubeflow user interface (UI). If th
-        the same value. You can edit the volume name, but if you later edit the 
+      * Name: The volume name is synced with the name of the notebook server,
+        and has the form `workspace-<server-name>`.
+        When you start typing the notebook server name, the volume name appears. 
+        You can edit the volume name, but if you later edit the 
         notebook server name, the volume name changes to match the notebook 
         server name.
       * Size: `10Gi`
-      * Mount path: `/home/jovyan`
       * Access mode: `ReadWriteOnce`. This setting means that the volume can be 
         mounted as read-write by a single node. See the 
         [Kubernetes documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for more details about access modes.
+      * Mount point: `/home/jovyan`
 
   * Alternatively, you can point the notebook server at an existing volume by 
-    specifying the name, mount path, and access mode for the existing volume.
+    specifying the name of the existing volume.
 
 1. *(Optional)* Specify one or more **data volumes** if you want to store and
   access data from the notebooks on this notebook server. You can add new
   volumes or specify existing volumes. Kubeflow provisions a 
   [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for each of your data volumes.
 
+1. *(Optional)* Specify one or more additional **configurations** as a list of
+  PodDefault labels. To make use of this option, you must create a
+  [PodDefault manifest](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md). In the PodDefault manifest, you can specify configurations
+  including volumes, secrets, and environment variables. 
+  Kubeflow matches the labels in the **configurations** field against
+  the properties specified in the PodDefault manifest. Kubeflow then injects 
+  these configurations into all the notebook Pods on this notebook server.
+  
+    For example, enter the label `addgcpsecret` in the **configurations** field
+    to match to a PodDefault manifest containing the following configuration:
+
+    ```
+    matchLabels:
+    addgcpsecret: "true"
+    ```
+
+    For indepth information on PodDefault usage, see the [admission-webhook
+    README](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md).
+
+1. *(Optional)* Change the setting for **enable shared memory**. The default is
+  that shared memory is enabled. Some libraries like PyTorch use shared memory
+  for multiprocessing. Currently there is no implementation in Kubernetes to 
+  activate shared memory. As a workaround, Kubeflow creates ab empty directory 
+  at `/dev/shm`. 
+
+1. *(Optional)* Specify one or more **extra resources** as a JSON string. The
+   JSON string must specify the value for one or more of the
+   `spec.containers[].resources.limits` options described in the [Kubernetes 
```

Comment:
The linked page seems to describe three options for the type of resource: `cpu`, `memory`, and `hugepages-<size>`. But the example you have added below seems to reserve GPU resources (`nvidia.com/gpu`) which doesn't match any of these (as far as I can tell). So I'm a bit confused as to what I can use in this section.

Also, let's say I do want to allocate extra cpu or memory in the **Extra Resources** section.  It seems like I already allocated these resources in steps 8 and 9 of this procedure, so what is the point or difference of doing that here?

Reply:
@lluunn Do you have an answer to Alec's question about how the JSON string `{"nvidia.com/gpu": 2}` fits into the `spec.containers[].resources.limits options`?

Reply:
This should be for GPU as cpu/memory can be set in previous steps as pointed out.
We can link to this page: https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/
There is an example usage of `nvidia.com/gpu: 1`

Reply:
Thanks both! I've clarified the section.